addons/l10n_at/data/template/account.fiscal.position-at.csv

```diff
@@ -11,10 +11,12 @@
 "","","","","","","","","chart_at_template_4000","chart_at_template_4100"
 "","","","","","","","","chart_at_template_4001","chart_at_template_4110"
 "","","","","","","","","chart_at_template_2000","chart_at_template_2100"
-"","","","","","","","","chart_at_template_5000","chart_at_template_5050"
+"","","","","","","","","chart_at_template_5010","chart_at_template_5051"
+"","","","","","","","","chart_at_template_5011","chart_at_template_5050"
```

Comment:
10 -> 51
11 -> 50

Looks kinda wrong...

Reply:
I can switch the codes of the 5051 and 5050 accounts?